examples/helm/kanister/kanister-mysql/README.md

```diff
@@ -61,12 +66,150 @@ $ helm install kanister/kanister-mysql -n my-release --namespace mysql-test \
+
+mysql> USE test;
+Database changed
+
+# Create "pets" table
+mysql> CREATE TABLE pets (name VARCHAR(20), owner VARCHAR(20), species VARCHAR(20), sex CHAR(1), birth DATE, death DATE);
+Query OK, 0 rows affected (0.02 sec)
+
+# Insert row to the table
+mysql> INSERT INTO pets VALUES ('Puffball','Diane','hamster','f','1999-03-30',NULL);
+Query OK, 1 row affected (0.01 sec)
+
+# View data in "pets" table
+mysql> SELECT * FROM pets;
++----------+-------+---------+------+------------+-------+
+| name     | owner | species | sex  | birth      | death |
++----------+-------+---------+------+------------+-------+
+| Puffball | Diane | hamster | f    | 1999-03-30 | NULL  |
++----------+-------+---------+------+------------+-------+
+1 row in set (0.00 sec)
+
+```
+
+## Protect the Application
+
+You can now take a backup of the MySQL data using an ActionSet defining backup for this application. Create an ActionSet in the same namespace as the controller.
+
+```bash
+$ kanctl create actionset --action backup --namespace kasten-io --blueprint my-release-kanister-mysql-blueprint --deployment mysql-test/my-release-kanister-mysql --profile mysql-test/mysql-test-profile
+actionset backup-rslmb created
+
+$ kubectl --namespace kasten-io get actionsets.cr.kanister.io
+NAME                 AGE
+backup-rslmb         1m
+
+# View the status of the actionset
+$ kubectl --namespace kasten-io describe actionset backup-rslmb
+```
+
+### Disaster strikes!
+
+Let's say someone accidentally deleted the test database using the following command:
+```bash
+# Connect to MySQL by running a shell inside MySQL's pod
+$ kubectl exec -ti $(kubectl get pods -n mysql-test --selector=app=my-release-kanister-mysql -o=jsonpath='{.items[0].metadata.name}') -n mysql-test -- bash
+
+# Drop the test database
+$ mysql> SHOW DATABASES;
++--------------------+
+| Database           |
++--------------------+
+| information_schema |
+| mysql              |
+| performance_schema |
+| sys                |
+| test               |
++--------------------+
+5 rows in set (0.00 sec)
+
+mysql> DROP DATABASE test;
+Query OK, 1 row affected (0.03 sec)
+
+mysql> SHOW DATABASES;
++--------------------+
+| Database           |
++--------------------+
+| information_schema |
+| mysql              |
+| performance_schema |
+| sys                |
++--------------------+
+4 rows in set (0.00 sec)
+
+```
+
+### Restore the Application
+
+To restore the missing data, you should use the backup that you created before. An easy way to do this is to leverage `kanctl`, a command-line tool that helps create ActionSets that depend on other ActionSets:
+
+```bash
+$ kanctl --namespace kasten-io create actionset --action restore --from "backup-llfb8"
+actionset restore-backup-62vxm-2hdsz created
+
+# View the status of the ActionSet
+kubectl --namespace kasten-io describe actionset restore-backup-62vxm-2hdsz
+```
+
+You should now see that the data has been successfully restored to MySQL!
```

Comment:
Should we say - `Once the ActionSet status is set to "complete", you can see that the data has been successfully restored to MySQL` just to make sure we wait until all the phases are completed?

Reply:
yes, will change this